code/workspaces/web-app/src/components/app/RequireAuth.js

```diff
@@ -5,9 +5,21 @@ import { Route } from 'react-router-dom';
 import { bindActionCreators } from 'redux';
 import { isEmpty } from 'lodash';
 import CircularProgress from '@material-ui/core/CircularProgress';
+import { withStyles } from '@material-ui/core';
 import getAuth from '../../auth/auth';
 import authActions from '../../actions/authActions';
 
+const circularProgressSize = 40; // Default size of component, used for position calculation
```

Comment:
Glad that this is now in the middle. Could we make it a bit bigger too?

Reply:
Should be possible, unfortunately not just by updating the value in the class that is applied. I need to have a look at finding the right bit of the MUI component to override to make it the desired size.